Tests/UnitTests/Scripts/tests.ts

```diff
@@ -416,7 +423,7 @@ if (hostPlatform !== "Unix") {
             };
 
             ws.onerror = (ev) => {
-                done(new Error(ev.message));
+                done(new Error(String(ev)));
```

Comment:
I tested this in a browser. It seems the event is just a generic event, so `ev.message` will be undefined. `String(ev)` will return `[object Event]` which isn't much better. The event itself does not seem to give more information. The console has a message when something fails. We should probably just say the websocket failed here.

```suggestion
                done(new Error("WebSocket failed"));
```